CMakePresets.json

```
@@ -3,14 +3,15 @@
   "$schema": "https://cmake.org/cmake/help/latest/_downloads/3e2d73bff478d88a7de0de736ba5e361/schema.json",
   "configurePresets": [
     {
-      "name": "ci-windows",
-      "displayName": "CI Windows",
+      "name": "ci-msvc",
+      "displayName": "CI MSVC",
       "generator": "Ninja",
       "cacheVariables": {
-        "CMAKE_COMPILE_WARNING_AS_ERROR": "1",
+        "CMAKE_COMPILE_WARNING_AS_ERROR": "ON",
         "BUILD_SHARED_LIBS": "OFF",
         "DD_TRACE_STATIC_CRT": "1",
-        "DD_TRACE_BUILD_TESTING": "1"
+        "DD_TRACE_BUILD_TESTING": "1",
```

Can we set this to `ON` for consistency?
```suggestion
        "DD_TRACE_BUILD_TESTING": "ON",
```